### DISCUSSION TITLE
SignServer CE 5.9.1.Final

### DISCUSSION THREAD

**Discussion by netmackan** [Keyfactor/signserver-ce#2] [Announcements]
Highlights
-------------

* Experimental Post-quantum Signing support
* Azure Key Vault Support
* DNSSEC Signing
* Custom Folder for Configuration
* Log4j Upgrade

Security Enhancements
---------------------

* Third-party Apache Santuario Library Upgrade (CVE-2021-40690)
* Fix for Cross-site Scripting Issue in Admin Web (CVE-2022-26494)

Release Notes
-------------------------
See full release notes and other announcements: 
https://doc.primekey.com/signserver/signserver-release-information/signserver-release-notes/signserver-community-5-9-1-release-notes

Visit the SignServer project on [signserver.org](https://signserver.org/).

<hr /><em>This discussion was created from the release <a href='https://github.com/Keyfactor/signserver-ce/releases/tag/v5.9.1.Final'>SignServer CE 5.9.1.Final</a>.</em>

**Comment by svenska-primekey** [comment]
Fantastic! Now I can update the Ansible playbook to pull SignServer from GitHub.

**Reply by netmackan** [reply]
Yeah, that is right, this is our first release since we moved the repo to GitHub.
